### PR TITLE
Add links

### PR DESCRIPTION
**Pull request by corochann**
Add related links

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -114,6 +114,26 @@ Use it at your own risk.
...
 Please see the [LICENSE](https://github.com/pfnet-research/chainer-chemistry/blob/master/LICENSE) file for details.
 
+## Links
+
+Links for Chainer Chemistry:
+
+ - Document: [https://chainer-chemistry.readthedocs.io](https://chainer-chemistry.readthedocs.io)
+ - Blog: [Release Chainer Chemistry: A library for Deep Learning in Biology and Chemistry](https://preferredresearch.jp/2017/12/18/chainer-chemistry-beta-release/)
+
+Links for other Chainer project:
```
Comment (delta2323) [inline README.md]: projects

File: README.md [hunk 2]
```diff
@@ -114,6 +114,26 @@ Use it at your own risk.
...
+
+ - Chainer: A flexible framework of neural networks for deep learning - [chainer/chainer](https://github.com/chainer/chainer)
```
Comment (delta2323) [inline README.md]: How about adding a link to titles (Chainer in this case)?

Comment (delta2323) [reply README.md]: Is it better to use the official website (https://chainer.org) as a destination URL?

File: README.md [hunk 3]
```diff
@@ -114,6 +114,26 @@ Use it at your own risk.
...
+ - ChainerRL: Deep reinforcement learning library built on top of Chainer - [chainer/chainerrl](https://github.com/chainer/chainerrl)
+ - ChainerCV: A Library for Deep Learning in Computer Vision - [chainer/chainercv](https://github.com/chainer/chainercv)
+ - ChainerMN: Scalable distributed deep learning with Chainer - [chainer/chainermn](https://github.com/chainer/chainermn)
+ - ChainerUI: User Interface for Chainer - [chainer/chainerui](https://github.com/chainer/chainerui)
+ - PaintsChainer: Line drawing colorization using chainer - [pfnet/PaintsChainer](https://github.com/pfnet/PaintsChainer)
```
Comment (delta2323) [inline README.md]: Which do you think it is better to use the demo URL (https://paintschainer.preferred.tech/) or github repository (https://github.com/pfnet/PaintsChainer) as a destination URL?

File: README.md (hunk 1 above)
Comment (delta2323) [reply README.md]: It think it is good to add CuPy, too.

File: README.md [hunk 4]
```diff
@@ -114,6 +114,26 @@ Use it at your own risk.
...
+ 
+If you are new with chainer, here is a tutorial to start with:
```
Comment (delta2323) [inline README.md]: If you are new to Chainer, ...

File: README.md (hunk 1 above)
Comment (corochann) [reply README.md]: noted.

File: README.md (hunk 2 above)
Comment (corochann) [reply README.md]: > adding a link to titles
What do you mean?

> Is it better to use the official website (https://chainer.org) as a destination URL?
May be we can put both (official page link as well as github link)

File: README.md (hunk 3 above)
Comment (corochann) [reply README.md]: Maybe we can put both.

File: README.md (hunk 4 above)
Comment (corochann) [reply README.md]: Noted.

**Comment by corochann** [comment]
Updated.
Is it better to put link on title for Chainer RL/CN/MN/UI?

**Comment by delta2323** [comment]
I imagined something like `[ChainerRL](https://github.com/chainer/chainerrl)` as it is more compact. But current code is also OK as it is a problem of tastes.

**Comment by corochann** [comment]
Then, I would like to use this format for now.
We can consider updating it again any time.

**Comment by delta2323** [comment]
LGTM

**Pull request merged**